apps/cards/tests/test_admin_dashboard.py

```
@@ -0,0 +1,23 @@
+
+
+pytestmark = pytest.mark.django_db
+
+
+class DashboardRFIDAttemptTests(TestCase):
+    def setUp(self):
+        user_model = get_user_model()
+        self.superuser = user_model.objects.create_superuser(
+            username="admin",
+            email="admin@example.com",
+            password="password",
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "password"

(S106)